coforma/swift-tech-challenge
# Details page

# Summary

Still need help with the styling and some formatting

- Add router for clicking on college cards to get to details page

Fixes #36 

## Type of change


# How To Test
- Export AWS credentials in your terminal
- Run `yarn dev`
- Open localhost:3000 and click on any college card
- Ensure it navigates to a new page with the college id as the path
- Ensure it shows information relevant to that college's data in the display

# Checklist:
